src/configure/helper/mustacheHelper.ts

```diff
@@ -220,12 +220,14 @@ export class MustacheHelper {
         }
 
         var result: Object = {};
-        Object.keys(mustacheObject).forEach(key => {
-            if (!!key && !!mustacheObject[key]) {
-                result[key] = MustacheHelper.renderObject(mustacheObject[key], view);
-            }
-        });
+        if (mustacheObject) {
```

Comment:
why this condition?

Reply:
It throws a null exception if null is passed as a mustacheObject.
Example scenario : changing dataSourceId of a parameter to null in overrides property.